### ISSUE TITLE
It is impossible to choose custom size in Choose Resize dialog with a tab key

### ISSUE THREAD

**Issue by CKFinderBOT** [ckfinder/ckfinder#252]
1. Go to samples/popups.html, click "Open Popup"
2. Focus a reasonably small image and press "Choose Resized" button. A dialog will appear with disabled "Large" option:

    ![screen shot 2015-08-26 at 16 34 46](https://cloud.githubusercontent.com/assets/545175/9496404/802bff34-4c10-11e5-926e-115c27fe2e1d.png)

Result: It is impossible to jump below the disabled radio with a tab key, which results in the following problems: I cannot select Custom and cannot reach Ok/Cancel buttons.

**Comment by CKFinderBOT** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
